src/main/java/use_cases/add_edit_category_use_case/CategoryUCI.java

```diff
@@ -131,4 +138,16 @@ public Category findCategory(ArrayList<Category> monthCategoryData, String name)
             if (Objects.equals(c.getName(), name)){
                 return c;}}
         throw new NoSuchElementException();}
+    /**
+     * Sets the information needed to create a CategoryCreatorInputData to call the createMonthObject method in CategoryFactory
+     * @return CategoryCreatorInputData MonthObjectFactoryInputData Object specifically used in CategoryFactory for the createMonthObject method.
+     */
+    private CategoryCreatorInputData setCategoryCreatorInfo(){
```

Comment:
These two private methods should return type MonthObjectFactoryInputData instead! This would better demonstrate Liskov's Substitution Principle.